scene/main/node.cpp

```diff
@@ -2391,7 +2391,7 @@ bool Node::is_in_group(const StringName &p_identifier) const {
 
 void Node::add_to_group(const StringName &p_identifier, bool p_persistent) {
 	ERR_THREAD_GUARD
-	ERR_FAIL_COND(!p_identifier.operator String().length());
+	ERR_FAIL_COND_MSG(!p_identifier.operator String().length(), "Tried to add node to a group with an empty name.");
```

Comment:
While we're here, let's fix the awkward test too :)

```suggestion
	ERR_FAIL_COND_MSG(p_identifier.is_empty(), "Tried to add node to a group with an empty name.");
```

Reply:
Non, ça fera une erreur. StringName ne fonctionne pas comme un String classique. Ca fonctionne si on fait String(p_identifier).is_empty()
Sinon, si p_identifier est égale à "", alors is_empty retournera false

Reply:
> While we're here, let's fix the awkward test too :)

Non, ça fera une erreur. StringName ne fonctionne pas comme un String classique. Ca fonctionne si on fait String(p_identifier).is_empty()
Sinon, si p_identifier est égale à "", alors is_empty retournera false